1-js/02-first-steps/03-strict-mode/article.md

```diff
@@ -1,89 +1,89 @@
+Așadar, cum se utilizează de fapt `use strict` în consolă?
 
-First, you can try to press `key:Shift+Enter` to input multiple lines, and put `use strict` on top, like this:
+În primul rând, puteți încerca să apăsați `key:Shift+Enter` pentru a introduce mai multe linii, și să puneți `use strict` deasupra, astfel:
 
 ```js
-'use strict'; <Shift+Enter for a newline>
-//  ...your code
-<Enter to run>
+'use strict'; <Shift+Enter pentru o linie nouă>
+//  ...codul tău
+<Enter pentru a executa>
 ```
 
-It works in most browsers, namely Firefox and Chrome.
+Funcționează în majoritatea browserelor, mai ales în Firefox și Chrome.
 
-If it doesn't, e.g. in an old browser, there's an ugly, but reliable way to ensure `use strict`. Put it inside this kind of wrapper:
+Dacă nu funcționează, de exemplu, într-un browser vechi, există o modalitate inestetică, dar fiabilă de a asigura `use strict`. Puneți-l în acest tip de înveliș:
 
 ```js
 (function() {
   'use strict';
 
-  // ...your code here...
+  // ...codul tău...
 })()
 ```
 
-## Should we "use strict"?
+## Ar trebui să folosim comanda "use strict"?
 
-The question may sound obvious, but it's not so.
+Întrebarea poate părea banală, dar nu este așa.
 
-One could recommend to start scripts with `"use strict"`... But you know what's cool?
+Cineva ar putea recomanda să începeți scripturile cu `"use strict"`... Dar știi ce este grozav?
 
-Modern JavaScript supports "classes" and "modules" - advanced language structures (we'll surely get to them), that enable `use strict` automatically. So we don't need to add the `"use strict"` directive, if we use them.
+JavaScriptul modern suportă "classes" ("clase") și "modules" ("module") - structuri avansate de limbaj (vom ajunge cu siguranță la ele), care permit `use strict` în mod automat. Astfel, nu este nevoie să adăugăm comanda `"use strict"`, dacă le folosim.
 
-**So, for now `"use strict";` is a welcome guest at the top of your scripts. Later, when your code is all in classes and modules, you may omit it.**
+**Deci, deocamdată `"use strict";` este binevenit în partea de sus a scripturilor dumneavoastră. Mai târziu, când tot codul dumneavoastră va cuprinde "classes" și "modules", îl veți putea omite.**
 
-As of now, we've got to know about `use strict` in general.
+Din acest moment, trebuie să știm despre `use strict` în general.
 
-In the next chapters, as we learn language features, we'll see the differences between the strict and old modes. Luckily, there aren't many and they actually make our lives better.
+În capitolele următoare, pe măsură ce vom învăța caracteristicile limbajului de programare, vom vedea diferențele dintre modul strict și cel vechi. Din fericire, nu sunt prea multe și, de fapt, ne fac viața mai ușoară.
 
-All examples in this tutorial assume strict mode unless (very rarely) specified otherwise.
+Toate exemplele din acest tutorial presupun modul strict, cu excepția cazului în care (foarte rar) se specifică altfel.
```

Comment:
```suggestion
Toate exemplele din acest tutorial presupun modul strict dacă nu (foarte rar) se specifică altfel.
```

Reply:
Îmi permit să-ți spun o poveste, Bogdan,

Eu am lucrat mulți ani în corporație, în asigurări, în domeniul juridic, practic am folosit cuvintele pentru a convinge adversarul și instituțiile statului că avem dreptate. Am lucrat cu foarte mulți oameni, diferite stiluri de abordare și de conducere.
Aveam o vorbă pentru acest stil, îl numeam ” we agree to disagree”.
Din acest motiv am făcut o reconversie către programare, sper să nu fie o alegere greșită.

Seara bună,

Reply:
Înțeleg... Codul de mai sus este doar o sugestie. Asta este doar o traducere simplă, nu presupune să îmbunătățim textul (decât dacă vrem să urmăm procedura și să facem îmbunătățirile la versiunea originală, apoi să o încorporăm în traducere).

Dacă adaug o virgulă și elimin parantezele sugestia arată cam așa:
```suggestion
Toate exemplele din acest tutorial presupun modul strict, dacă nu se specifică altfel.
```

La sensul ăsta mă gândeam. 
`cu excepția cazului în care (foarte rar) se specifică altfel.` s-ar fi tradus înapoi în engleză în: 
`with the exception of when (very rarely) it is specified otherwise.` sau ceva similar.
de asta `dacă nu se specifică altfel.` părea că se potrivește mai bine.
Dar e posibil să nu am dreptate.